move_group/src/default_capabilities/move_action_capability.cpp

```diff
@@ -169,7 +170,6 @@ bool move_group::MoveGroupMoveAction::planUsingPlanningPipeline(const planning_i
 {
   setMoveState(PLANNING);
 
-  planning_scene_monitor::LockedPlanningSceneRO lscene(plan.planning_scene_monitor_);
```

Comment:
Are you sure this is not needed?
@isucan added that code in 45067d4d in 2013 and it wasn't mentioned anywhere in the function back then either...